.goreleaser.yml

```diff
@@ -95,6 +95,7 @@ dockers: # https://goreleaser.com/customization/docker/
       - "--label=org.opencontainers.image.title={{.ProjectName}}"
       - "--label=org.opencontainers.image.revision={{.FullCommit}}"
       - "--label=org.opencontainers.image.version={{.Version}}"
+      - "--build-arg=VERSION={{ .Version }}"
```

Comment:
Am I correct, this implies we always need a cnquery image of the same version?

So, e.g., a cnspec only bugfix like v7.3.1 wouldn't be possible without previously generating a v7.3.1 cnquery image?

Reply:
yes, it's seems like it. But from what I understood when we install cnquery and cnspec, they will be part of the same installer (e.g. the MSI for windows). If that is the case, I think this should be fine as well because we will always release them together?

My guess is that we can work with this for now and if we run into troubles releasing where we don't want to release both cnspec and cnquery, we can re-evaluate

Reply:
I agree let`s see if we run into the issue where we need to use a different version.